tests/test_init.py

```
@@ -1854,3 +2119,55 @@ def async_update_device(self, *a, **k) -> Never:
+    )
+    second_filesystem_entity = MagicMock(
+        entity_id="sensor.router_telemetry_filesystems_slash_dev_slash_sdb1",
+        unique_id="router_telemetry_filesystems_slash_dev_slash_sdb1",
+    )
+
+    hass = ph_hass
+    hass.config_entries = MagicMock()
+    hass.config_entries.async_update_entry = MagicMock(return_value=True)
+
+    entity_registry = MagicMock()
+    entity_registry.async_entries_for_config_entry = lambda registry, config_entry_id: [
```

Ruff (0.15.18)

[warning] Unused lambda argument: `registry`

(ARG005)